core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2859,6 +2859,7 @@ class ViewMapping< Traits ,
 template< class DstTraits , class SrcTraits >
 class ViewMapping< DstTraits , SrcTraits ,
   typename std::enable_if<(
+    !(std::is_same<typename SrcTraits::array_layout, LayoutStride>::value) &&//Added to have a new specialization for SrcType of LayoutStride
```

Comment:
Some compilers might prefer that you leave a space between `&&` and `//`.  Ditto for `*/` and `//` above.  Something about trigraphs etc.